compiler/ballerina-backend-jvm/src/main/ballerina/compiler_backend_jvm/jvm_terminator_gen.bal

```diff
@@ -562,19 +591,73 @@ type TerminatorGenerator object {
         self.genYieldCheck(fpCall.thenBB, funcName);   
     }
 
+    function genWrkSendIns(bir:WrkSend ins, string funcName) {
```

Comment:
Shall we use worker text instead of wrk?

Reply:
+1, I'll do this change in a subsequent PR.